lib/streamlit/web/server/media_file_handler.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2018-2022 Streamlit Inc.
```

Comment:
This whole file is just a cut-and-paste from routes.py (no MediaFileHandler behavior changes).